utility/src/test/java/org/oppia/android/util/locale/DisplayLocaleImplTest.kt

```diff
@@ -49,12 +49,12 @@ class DisplayLocaleImplTest {
   @Inject
   lateinit var wrapperChecker: TestOppiaBidiFormatter.Checker
 
-  @Inject
   lateinit var context: Context
 
   @Before
   fun setUp() {
     setUpTestApplicationComponent()
+    context = InstrumentationRegistry.getInstrumentation().targetContext
```

Comment:
Why is this needed vs. the injection? The injected context should match the target context being retrieved here.

Reply:
Reverted.